server/services/EventHandlers.js

```diff
@@ -26,5 +26,6 @@ const registerEventHandlers = () => {
     subscribe("admin-verification", handleVerifyCaptureProcessed);
 }
 
-module.exports = registerEventHandlers;
-
+module.exports = {handleVerifyCaptureProcessed, registerEventHandlers};

```

Comment:
What we really want to export in place of `handleVerifyCaptureProcessed` here is a dictionary of  event types to event handlers that can be used at the point of replaying the payloads that are in `received` status.  Say if we have two different domain event that has different payloads. Say one is 'AdminVerification' event and another is 'TokenIssued' event. In these cases we want to use different functions to process them. For that we need the ability to lookup the required function by using the event `type` attribute.